pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -118,31 +122,32 @@ def _validate_condition_metric(self, logs):
         return True
 
     @property
-    def monitor_op(self):
+    def monitor_op(self) -> Callable:
         return self.mode_dict[self.mode]
 
-    def on_save_checkpoint(self, trainer, pl_module, checkpoint: Dict[str, Any]) -> Dict[str, Any]:
+    def on_save_checkpoint(self, trainer: 'Trainer', pl_module: 'LightningModule',
+                           checkpoint: Dict[str, Any]) -> Dict[str, Any]:
         return {
             'wait_count': self.wait_count,
             'stopped_epoch': self.stopped_epoch,
             'best_score': self.best_score,
             'patience': self.patience
         }
 
-    def on_load_checkpoint(self, callback_state: Dict[str, Any]):
+    def on_load_checkpoint(self, callback_state: Dict[str, Any]) -> None:
         self.wait_count = callback_state['wait_count']
         self.stopped_epoch = callback_state['stopped_epoch']
         self.best_score = callback_state['best_score']
         self.patience = callback_state['patience']
 
-    def on_validation_end(self, trainer, pl_module):
+    def on_validation_end(self, trainer: 'Trainer', pl_module: 'LightningModule') -> None:
         from pytorch_lightning.trainer.states import TrainerState
         if trainer.state != TrainerState.FITTING or trainer.sanity_checking:
-            return
+            return None
 
         self._run_early_stopping_check(trainer)
```

Comment:
`return None` is excessive - why not `return` only?

Reply:
In a couple of cases where there was `return` but the return type wasn't only `None`, but some optional return, mypy complained that a plain return wouldn't be sufficient but it had to be explicit, so I did it like this all the time :)